priv/repo/migrations/20240408124205_add_tenants_database_index.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Trento.Repo.Migrations.AddTenantsDatabaseIndex do
+  use Ecto.Migration
+
+  def up do
```

Comment:
Simple question: what is this for?
I mean you create an index to remove it later on